src/content/docs/query-your-data/explore-query-data/dashboards/add-custom-visualizations-your-dashboards.mdx

```diff
@@ -26,7 +26,7 @@ The process for creating a visualization is covered in the guide, [Build a custo
 
 You can add a visualization to a new or existing dashboard.
 
-1. From [New Relic](https://one.newrelic.com), in the top right, click the **Apps** button, and then on the **Apps** page, click **Custom Visualizations**. Hint: if you don't see the **Custom Visualizations** tile, use the search to locate it.
+1. From **[one.newrelic.com](https://one.newrelic.com)** in the top right, click the **Apps** button, and then on the **Apps** page, click **Custom Visualizations**. Hint: if you don't see the **Custom Visualizations** tile, use the search to locate it.
```

Comment:
I'd add a comma: From **[one.newrelic.com](https://one.newrelic.com)**,  in the top right....